tests/python/pants_test/tasks/test_changed_target_integration.py

```diff
@@ -1,149 +0,0 @@
-# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Do you think `changed_integration_test.py` covers all the cases of this? It seems like yes to me.